src/Task.gren

```diff
@@ -145,70 +150,8 @@ If it fails, the whole thing fails!
 
 -}
 map2 : (a -> b -> result) -> Task x a -> Task x b -> Task x result
-map2 func taskA taskB =
-    taskA
-        |> andThen
-            (\a ->
-                taskB
-                    |> andThen (\b -> succeed (func a b))
-            )
-
-
-{-| -}
-map3 : (a -> b -> c -> result) -> Task x a -> Task x b -> Task x c -> Task x result
-map3 func taskA taskB taskC =
```

Comment:
I actually thought `map3` was fine 😅

Reply:
Oh sorry, I misread your original comment.

On a side note, do you think you'll do a similar thing with `Maybe`, `Result`, `Random` etc for consistency? They each have `map5`.

Reply:
> On a side note, do you think you'll do a similar thing with ...

I've been meaning to, yes. :)